source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,73 @@ Support for Mattermost Server v9.11 [Extended Support Release](https://docs.matt
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
 ```
 
+(release-v10.6-feature-release)=
+## Release v10.6 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release day: 2025-03-14**
+
+### Important Upgrade Notes
+ - Support for PostgreSQL v11 and v12 have been removed. The new minimum PostgreSQL version is v13+. See the :ref:`minimum supported PostgreSQL version policy <install/prepare-mattermost-database:minimum supported version policy>` documentation for details.
+ - Migration times: On a system with 12M posts, and 1M fileinfo entries, the migration takes 15s. It is also non-blocking. Note that there is no migration for MySQL installation because this optimization is only applicable for PostgreSQL.
```

Comment:
```suggestion
 - Migration times: On a system with 12M posts, and 1M fileinfo entries, the migration takes 15s. This migration is non-blocking. Note that there is no migration for MySQL deployments because this optimization is only applicable for PostgreSQL.
```